package-lock.json

```diff
@@ -4,106 +4,100 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "7zip-bin": {
```

Comment:
What's with this entire diff? You haven't changed package.json even :-O

Reply:
Umh, looks like my `npm update` changed things... Removing this file from the PR

Reply:
Removed